tox.ini

```
@@ -1,49 +1,45 @@
 [tox]
-envlist = py38-django{22,30,31,32}
+envlist = py38-django{32,40}
 skipsdist = true
 
 [pytest]
-DJANGO_SETTINGS_MODULE = analytics_dashboard.settings.test
+django_settings_module = analytics_dashboard.settings.test
 
 [testenv]
-passenv =
-	API_SERVER_URL
-	API_AUTH_TOKEN
-	LMS_HOSTNAME
-	LMS_PASSWORD
-	LMS_USERNAME
-	ENABLE_AUTO_AUTH
-	ENABLE_OAUTH_TESTS
-	ENABLE_ERROR_PAGE_TESTS
-	DISPLAY_LEARNER_ANALYTICS
-	ENABLE_COURSE_LIST_FILTERS
-	ENABLE_COURSE_LIST_PASSING
-	COVERAGE_DIR
-	DISPLAY
-	SELENIUM_BROWSER
-	BOKCHOY_HEADLESS
-deps =
-    django22: Django>=2.2,<2.3
-    django30: Django>=3.0,<3.1
-    django31: Django>=3.1,<3.2
-    django32: -r requirements/django.txt
+passenv = 
+    API_SERVER_URL
+    API_AUTH_TOKEN
+    LMS_HOSTNAME
+    LMS_PASSWORD
+    LMS_USERNAME
+    ENABLE_AUTO_AUTH
+    ENABLE_OAUTH_TESTS
+    ENABLE_ERROR_PAGE_TESTS
+    DISPLAY_LEARNER_ANALYTICS
+    ENABLE_COURSE_LIST_FILTERS
+    ENABLE_COURSE_LIST_PASSING
+    COVERAGE_DIR
+    DISPLAY
+    SELENIUM_BROWSER
+    BOKCHOY_HEADLESS
+deps = 
+    django32: Django>=3.2,<4.0
```

You should instead update the `requirements/django.txt` and use that here. 
```suggestion
    django32: -r requirements/django.txt
```